src/tests/helpers.js

```diff
@@ -127,28 +127,6 @@ export const mockRadioInput = {
   },
   type: 'string',
 };
-export const mockRadioInputOptional = {
```

Comment:
note: This wasn't deleted, it was just moved to line 866 for better readability.